README.md

```diff
@@ -21,13 +21,36 @@
 </a>
 
 # Fingerprint Pro Cloudflare worker
-The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
+
+[Fingerprint](https://fingerprint.com/) is a device intelligence platform offering 99.5% accurate visitor identification.
+
+Fingerprint Pro Cloudflare Integration is responsible for
+
+- Proxying download requests of the latest Fingerprint Pro JS Agent between your site and Fingerprint CDN.
+- Proxying identification requests and responses between your site and Fingerprint Pro's APIs.
+
+This [improves](https://dev.fingerprint.com/docs/cloudfront-proxy-integration#the-benefits-of-using-the-cloudfront-integration) both accuracy and reliability of visitor identification and bot detection on your site.
+
+## Requirements
+
+### Usage Requirements
+
+- Credentials from Cloudflare
+  - [Cloudflare Account ID](https://dev.fingerprint.com/docs/cloudflare-integration#cloudflare-account-id)
+  - [Cloudflare API Token](https://dev.fingerprint.com/docs/cloudflare-integration#cloudflare-api-token)
+- FingerprintPro Account
+
+By just following the wizard with your credentials it's easy to setup. Please see the [Setup Process](#setup-process) section
```

Comment:
@Orkuncakilkaya These are not the requirements listed in the docs. Getting Cloudflare account ID and token is covered in the guide, therefore these are not requirements. Requirements or pre-requisites are "stuff the guide/tool assumes you already have before you start", not "stuff the guide helps you set up".

```suggestion
* [Fingerprint Pro account](https://dashboard.fingerprint.com/signup) with the _Owner_ role assigned.
* A website served by Cloudflare. For maximum accuracy benefits, your website should be [proxied by Cloudflare](https://developers.cloudflare.com/dns/manage-dns-records/reference/proxied-dns-records/) (not DNS-only). 

See the [Cloudflare integration guide](https://dev.fingerprint.com/docs/cloudflare-integration#setup) for more details. 

```